what4-serialize.cabal

```diff
@@ -15,6 +15,7 @@ Description:
 library
   build-depends:
     base >= 4.8 && < 5,
+    bv-sized >= 1.0.0,
```

Comment:
It isn't the most popular opinion, but I'd appreciate an upper bound on this (whatever is reasonable for indicating you would make a breaking API change)